app/views/admin/project_logs/_project_log.html.slim

```diff
@@ -0,0 +1,42 @@
+- log = project_log
+- step = log.project_step
+
+.log
+  header
+    .infos
+      .item
+        i.fa.fa-calendar-o>
+        = l log.date
+      .item
+        i.fa.fa-user>
+        a href=admin_person_path(log.agent) = log.agent_name
+      .item
+        i.fa.fa-commenting-o>
+        = log.progress_metric
+
+    .step-name 
+      a href=(admin_loan_path(step.project) + "/timeline-list#step-#{step.id}") 
+        = project_step_icon(step)
+        = step.summary
+
+  - if log.summary
+    span.summary = log.summary
+
+  - if [log.details, log.additional_notes, log.private_notes].any?(&:present?)
+    a data-expands="more-#{log.id}" = t(:more)
+    a data-hides="more-#{log.id}" = t(:less)
+
+    div data-expandable="more-#{log.id}"
+      - if log.details.present?
+        .log-info
+          = simple_format log.details.to_s
```

Comment:
Brendan asked about preserving line breaks in the logs. (Now he wants markdown, or something like CKeditor!)